cli/azd/pkg/exec/cmdtree_windows.go

```diff
@@ -79,3 +125,35 @@ func (o *CmdTree) Kill() {
 		log.Printf("failed to terminate job object %d: %s\n", o.jobObject, err)
 	}
 }
+
+// buildCmdCmdLine builds the command line for`cmd.exe` to have it run command or list of commands. When cmd is non empty
+// it is treated as the command to run and args is the array of arguments for cmd. Otherwise, each arg is treated as an
+// individual command, and joined with &&.  Each individual command is quoted.
+func buildCmdCmdLine(cmd string, args []string) string {
+	var cmdLine strings.Builder
+	cmdLine.WriteString("/c ")
+	cmdLine.WriteString(`"`)
+	if cmd == "" {
+		for idx, arg := range args {
+			if idx != 0 {
+				cmdLine.WriteString("  &&  ")
+			}
+			cmdLine.WriteString(`"`)
```

Comment:
Looks like the quoting may be causing a [test ](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3073923&view=logs&jobId=a24bcfce-df5c-5f85-8999-2d93251fa3c8&j=a24bcfce-df5c-5f85-8999-2d93251fa3c8&t=aefa19dc-d857-50c1-f3d9-ed0b8a97a0d4)to fail? Could be due to incorrect usage.